src/ai/backend/client/cli/scp.py

```diff
@@ -9,19 +9,19 @@
 @click.argument("dest",  type=str, default="work@localhost", metavar='DEST')
 @click.option('-p', '--port',  type=str, metavar='PORT', default=9922,
               help="the port number is for localhost")
-@click.option('-r',  metavar="RECURSIVE", default=False, is_flag=True,
-              help="recursive option to process directories")
-def scp(sess_name, src, dest, port, r):
+@click.option('-r',  '--recursive', default=False, is_flag=True,
+              help="recursive flag option to process directories")
+def scp(sess_name, src, dest, port, recursive):
     """SCP client to Backend.AI SSH server.
 
     \b
-    It is assumed that first connection is made through `backend.ai ssh sess_name`
+    First connection is made through `backend.ai ssh sess_name`
     which starts SSH Server.
     SESS_NAME: Name of the running session.
     SRC: file or directory which you want to move
     Dest: file or directory destination. Ex) work@localhost:tmp/
     Port: Port num where ssh is opened
-    R: Option to download/ upload directories
+    Recursive: Option to download/ upload directories
```

Comment:
When Click generates the help text, optional arguments are rendered from their help arguments so this part is duplicate. Put only the positional arguments with upper-cased meta-variables.